src/context/AuthContext.tsx

```diff
@@ -1,51 +1,63 @@
-import { createContext, useContext, useState } from 'react';
+import { createContext, useState } from 'react';
 import type { ReactNode } from 'react';
 
 type UserRole = 'employer' | 'employee' | null;
 
-interface AlarmType {
-  id: string                                    // 공고 id
-  status: 'accepted' | 'rejected';              // 공고 지원 상태
-  restaurantName: string;                       // 음식점 이름 
-  startsAt: string;                             // 공고 시작 시간 (ISO 8601 문자열)
-  workHour: number;                             // 근무 시간 (시간 단위) 
-  createdAt: string;                            // 알림 생성 시간 (ISO 8601 문자열) 
+interface AlarmType { 
+  item: {
+    id: string;                                 // 가게 id
+    createdAt: string;                          // 생성 일시 (ISO 8601 문자열) 
+    result: 'accepted' | 'rejected';            // 결과 상태 
+    read: boolean;                              // 읽음 여부 
+    shop: {
+      item: {
+        name: string;                           // 가게 이름
+      };
+    };
+    notice: {
+      item: {
+        startsAt: string;                       // 근무 시작 시간 (ISO 8601 문자열) 
+        workhour: number;                       // 근무 시간 (시간 단위) 
+      };
+    };
+  };
+}
+
+interface AlarmInfo {
+  count: number;
+  items: AlarmType[];
 }
 
 interface AuthContextType {
   isLoggedIn: boolean;                          // 로그인 여부
   role: UserRole;                               // 알바님, 사장님 구분
-  alarms: AlarmType[];                          // 알림 리스트
+  alarms: AlarmInfo;                            // 알림 리스트
   logout: () => void;                           // 로그아웃 함수
 }
 
 const defaultAuthContext: AuthContextType = {
   isLoggedIn: false,
   role: null,
-  alarms: [],
+  alarms: {count: 0, items:[]},
   logout: () => {},
 };
 
-const AuthContext = createContext<AuthContextType>(defaultAuthContext);
+export const AuthContext = createContext<AuthContextType>(defaultAuthContext);
 
 export function AuthProvider({ children }: { children: ReactNode }) {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
   const [role, setRole] = useState<UserRole>(null);
-  const [alarms, setAlarms] = useState<AlarmType[]>([]);
+  const [alarms, setAlarms] = useState<AlarmInfo>({count: 0, items:[]});
```

Comment:
```suggestion
  const [alarms, setAlarms] = useState<AlarmInfo>(defaultAuthContext.alarms);
```
💬 이런 식으로도 사용이 가능할거 같아요?